pkg/kubelet/dockershim/docker_service.go

```diff
@@ -257,16 +257,19 @@ func NewDockerService(config *ClientConfig, podSandboxImage string, streamingCon
 	ds.network = network.NewPluginManager(plug)
 	klog.InfoS("Docker cri networking managed by the network plugin", "networkPluginName", plug.Name())
 
+	dockerInfo, err := ds.client.Info()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to execute Info() call to the Docker client")
```

Comment:
this is a change of behavior. If docker is slow to respond, kubelet may fail to start now? I don't think it's a critical change, just leaving this comment for other reviewers to pay attention.

Reply:
agreed, I called this out here: https://github.com/kubernetes/kubernetes/pull/104287#issuecomment-963710205.  There are other calls to the docker during this time that could cause a failure, so we should be confident that docker is running

Reply:
yep, I see it now =) Sorry missed it